src/main/java/org/folio/oaipmh/helpers/GetOaiIdentifiersHelper.java

```diff
@@ -57,9 +59,10 @@
       }
 
       HttpClientInterface httpClient = getOkapiClient(request.getOkapiHeaders());
+      final String instanceEndpoint = storageHelper.buildRecordsEndpoint(request,isDeletedRecordsEnabled(request, REPOSITORY_DELETED_RECORDS));
```

Comment:
It seems to me that you haven't formatted the code properly. There is no whitespace after the comma. Such lines make it a bit hard to read the code.